### ISSUE TITLE
CC API v3 Proposals

### ISSUE THREAD

**Issue by chenl23** [chenl23/cfar-proposals#6]
### Proposal Name
CC API v3 Proposals

### TL;DR Summary
As part of the v3 acceleration initiative we announced last October, we have written a number of proposals for the remaining resources that will be exposed on the v3 API.

If you work on a project the uses the CC API, we would love your feedback on these proposals (and on your experience with the v3 API in general). Please feel free to comment on the docs, respond to this email, or message us in the #v3-acceleration-team channel on the Cloud Foundry Slack.

### Proposal URL
https://docs.google.com/document/d/1g48YPBfwXT8kNrJYroBNkhlCtROe53INEEzvhQaAUOI/edit

### Point of Contact
Greg Cobb
